frontend/src/sass/pages/dashboard.scss

```diff
@@ -28,6 +28,10 @@ body {
   background-color: rgba(234, 237, 241, 1);
 }
 
+input {
+  color: #000;
```

Comment:
I'm fine with this as long as there are no knock-on effects of making this change global to the whole dashboard.